addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/thing-types.xml

```diff
@@ -25,6 +25,11 @@
 				<description>Port of the Onkyo to control</description>
 				<default>60128</default>
 			</parameter>
+			<parameter name="udn" type="text">
+				<label>Unique Device Name</label>
+				<description>The UDN identifies the Zone Player.</description>
```

Comment:
Is "Zone Player" a copy&paste error from Sonos or is this real vocabulary for Onkyo? Below, you have called it "Onkyo AVR".